pkgs/top-level/all-packages.nix

```diff
@@ -290,9 +290,9 @@ with pkgs;
 
   _0x =  callPackage ../tools/misc/0x { };
 
-  atuin = callPackage ../tools/misc/atuin {
-    inherit (darwin.apple_sdk.frameworks) AppKit Security SystemConfiguration;
-  };
+  #atuin = callPackage ../tools/misc/atuin {
```

Comment:
Why not just remove these lines? We can always recover them from the git history if needed.